src/custom/theme/index.tsx

```diff
@@ -0,0 +1,105 @@
+import { DefaultTheme, ThemeProvider as StyledComponentsThemeProvider, createGlobalStyle, css } from 'styled-components'
+import React, { useMemo } from 'react'
+
+import { Colors } from 'theme/styled'
+import {
+  colors as colorsUniswap,
+  theme as themeUniswap,
+  FixedGlobalStyle as FixedGlobalStyleUniswap,
+  ThemedGlobalStyle as ThemedGlobalStyleUniswap
+} from '../../theme'
+import { useIsDarkMode } from 'state/user/hooks'
+
+export { TYPE } from '../../theme'
+export * from '../../theme/components'
+
+export function colors(darkMode: boolean): Colors {
+  return {
+    ...colorsUniswap(darkMode),
+
+    // ****** base ******
+    // white: '#FFFFFF,
+    // black: '#000000',
+
+    // ****** text ******
+    // text1: darkMode ? '#FFFFFF' : '#000000',
+    text2: darkMode ? '#DCDCDC' : '#565A69',
```

Comment:
I think it would be easier if we did
```ts
const  overrides = darkMode ? {
  text2: '#DCDCDC',...
} : {
  text2: '#565A69',...
}
```

rather than have a hundred ternaries

Reply:
I think so too, but in this case, I would kind of prefer, at least for now, be close to uniswap. Note there's copy&past&commenting involved

Reply:
I think, for non-typescript project it might make sense, so u don't forget one color in one of the branches. In TS doesn't make sense. However, right now it weights more for me, keeping it close to Uni